### DISCUSSION TITLE
remove unused style tag css issue in react js + next js

### DISCUSSION THREAD

**Discussion by naveennsit** [vercel/next.js#21160] [Help]
**What version of Next.js are you using?**

10.0.1

**What version of Node.js are you using?**

10.0.0

**What browser are you using?**

chrome

**What operating system are you using?**

macos

**How are you deploying your application?**

next start

**Describe the Bug**

can we removed  unused style tag css issue in react js + next js .Here is my code
https://codesandbox.io/s/floral-hill-x176r?file=/postcss.config.js
I used like that

     <div>
          Hello World.{" "}
          <Link href="/about">
            <a className="red">About</a>
          </Link>
          <style jsx>{`
            .container {
              margin: 50px;
            }
            p {
              color: blue;
            }
            .red {
              color: red;
            }
            .blue {
              color: red;
            }
          `}</style>
        </div>

I only used `red` class so it will remove all other class .But in my case it is not removing.

I am using `postcss` plugin to remove unused css.but it is not working.[![enter image description here][1]][1] 

  [1]: https://i.stack.imgur.com/P1Fuo.png

**Expected Behavior**

I only used `red` class so it will remove all other class .But in my case it is not removing.

**To Reproduce**

https://codesandbox.io/s/floral-hill-x176r?file=/postcss.config.js

**Comment by liangwn** [comment]
您好，您发的邮件我已收到，尽快给您回复！
